### PR TITLE
Remove double-negative in CORS-preflight fetch credentials tags

### LINKED ISSUE #279: Credentials flag text in "CORS-preflight fetch" is unnecessarily convoluted
Step 7.12 of CORS-preflight fetch includes this text: `False if request's credentials mode is not "include", and true otherwise`. As far as I can tell, this is equivalent to `True if request's credentials mode is "include", false otherwise`.

### PR DESCRIPTION
**Pull request by jeffcarp**
Resolves #279 

First pull request - happy to change anything. Thanks for adding the `good first bug` label to issues!

### CODE REVIEW

File: Overview.src.html [hunk 1]
```diff
@@ -3406,9 +3406,9 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
      <dd><var>max-age</var>
 
      <dt><span title=concept-cache-credentials>credentials</span>
-     <dd>False if <var>request</var>'s
-     <span title=concept-request-credentials-mode>credentials mode</span> is not
-     "<code title>include</code>", and true otherwise
+     <dd>True if <var>request</var>'s
+     <span title=concept-request-credentials-mode>credentials mode</span>
+     is "<code title>include</code>", false otherwise
```
Comment (annevk) [inline Overview.src.html]: Please keep "and" here and below.

And feel free to add your name to the list of Acknowledgements.

**Comment by annevk** [comment]
Oh also, one other minor request I had is if you could stick to the wrapping guidelines. This text is old so might not conform to it yet, but it would be great to make it so as part of this commit. I'm happy to clean that up as part of merging however. Just a thought.

**Comment by jeffcarp** [comment]
Added "and" and added myself to the acknowledgements, thanks! 👍

I'm happy to make the changes to conform to the wrapping guidelines but I can't exactly figure out what needs to be changed.

**Comment by annevk** [comment]
Thank you, landed as https://github.com/whatwg/fetch/commit/f9a540bd002679bfa238f2fddf5543e7ac9b0d11.

**Comment by jeffcarp** [comment]
🎆 🎉 yay thank you!

**Pull request closed without merging**